sdk/typescript/src/cryptography/signature.ts

```diff
@@ -9,12 +9,14 @@ export type SignatureScheme = 'ED25519' | 'Secp256k1' | 'Secp256r1' | 'MultiSig'
 /**
  * Pair of signature and corresponding public key
  */
-export type SignaturePubkeyPair = {
+export type SerializeSignatureInput = {
```

Comment:
Moved `SignaturePubkeyPair` to `cryptography/utils.ts`m and renamed this version here.  This type was being used for multiple things, that probably shouldn't have been the same type